Home2/Program.cs

```diff
@@ -6,6 +6,15 @@ internal class Program
     {
         static void Main(string[] args)
         {
+            Task1();
+            Task2();
+            Task3();
+            Task4();
+        }
+        public static void Task1()
+        {
+
```

Comment:
Please, remove redundant spaces to keep code in the following way:
{
some code
}